Tests/UnitTests/Core/Propagator/EigenStepperTests.cpp

```diff
@@ -968,7 +969,7 @@ BOOST_AUTO_TEST_CASE(step_extension_vacmatvac_test) {
 // Test case a). The DenseEnvironmentExtension should state that it is not
 // valid in this case.
 BOOST_AUTO_TEST_CASE(step_extension_trackercalomdt_test) {
-  double rotationAngle = M_PI * 0.5;
```

Comment:
How many different version of this are int he code :-P

Reply:
Now, there is just `pi / 2.` :)